docs/_posts/2017-06-14-jekyll-3-5-0-released.markdown

```diff
@@ -0,0 +1,38 @@
+---
+title: 'Jekyll turns 3.5, oh my!'
+date: 2017-06-15 17:32:32 -0400
+author: parkr
+version: 3.5.0
+categories: [release]
+---
+
+Good news! Nearly 400 commits later, Jekyll 3.5.0 has been released into
+the wild. Some new shiny things you might want to test out:
+
+- Jekyll now uses Liquid 4, the latest!
```

Comment:
https://github.com/Shopify/liquid/blob/master/History.md#400--2016-12-14--branch-4-0-stable it seems the main feature is dropped support for ruby 2.0?